fonduer/utils/udf.py

```diff
@@ -7,7 +7,7 @@
 try:
     from IPython import get_ipython
 
-    if "IPKernelApp" not in get_ipython().config:
+    if get_ipython() is None or "IPKernelApp" not in get_ipython().config:
```

Comment:
if `get_ipython()` is None, it should trigger the `AttributeError` that is caught, right? Did you see some different error?

Reply:
Yeah, AttributeError works fine.

Reply:
Yeah, I AttributeError works fine.